styles.css

```diff
@@ -26,4 +26,12 @@ li a {
   padding: 20px 16px;
   text-decoration: none;
   font-family: "Courier New", Courier, monospace
+}
+
+section {
+  max-height: 800px;
+  max-width: 800px;
+  text-align: justify;
```

Comment:
To address your comment, my opinion is that it might look better to set a max-width on your content and then to center your content section. See https://getmdl.io/templates/blog/index.html for an example.

However, please open an Issue and assign it to me if you wish to discuss further to avoid scope creep (https://en.wikipedia.org/wiki/Scope_creep) on this PR.